open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -39,22 +54,20 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
 
```

Comment:
Need an apostrophe in "... that contains the tenant's serverName"

Also the environment variable for configuration presentation server is really called EGERIA_PRESENTATIONSERVER_SERVER_<tenant_name> 

It would be really nice to simplify the env variable name - it is kind of ugly.